eagleEvents/templates/layout.html.j2

```diff
@@ -8,36 +8,43 @@
     <link rel="stylesheet" href="/static/style.css">
     <link rel="stylesheet" href="/static/normalize.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
-    <script       src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script>
+    $(".nav a").on("click", function(){
+    $(".nav").find(".active").removeClass("active");
+    $(this).parent().addClass("active");});
+    </script>
 
     {% block head_content %}
       <title>Eagle Events Layout</title>
     {% endblock %}
   </head>
   <body class="main">
       {% block nav_content %} 
-        <nav class="navbar navbar-default">
-          <div class="container-fluid">
-            <div class="navbar-header">
-              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
-                <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
-                <span class="icon-bar"></span>                        
-              </button>
-              <a class="navbar-brand" href="event.html.j2">Eagle Event Planner</a>
-            </div>
-            <div class="collapse navbar-collapse" id="myNavbar">
-              <ul class="nav navbar-nav">
-                <li><a href="/listEvents">Events</a></li>
-                <li><a href="/listCustomers">Customers</a></li>
-                <li><a href="event-planner.html.j2">Event Planners</a></li>
-              </ul>
-              <ul class="nav navbar-nav navbar-right">
-                <li><a href="#"><span class="glyphicon glyphicon-log-in"></span> Logout</a></li>
-              </ul>
-            </div>
-          </div>
-        </nav>
+
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>                        
+      </button>
+      <a class="navbar-brand" href="event.html.j2">Eagle Event Planner</a>
+    </div>
+    <div class="collapse navbar-collapse" id="myNavbar">
+      <ul class="nav navbar-nav">
+        <li class="active"><a href="/listEvents">Events</a></li>
+        <li><a href="/listCustomers">Customers</a></li>
+        <li><a href="event-planner.html.j2">Event Planners</a></li>
```

Comment:
This should be the corresponding route for event planners

Reply:
shouldn't this just be like this?
              `<ul class="nav navbar-nav">`
               ` <li><a href="/listEvents">Events</a></li>`
                `<li><a href="/listCustomers">Customers</a></li>`
               ` <li><a href="/listEventPlanners">Event Planners</a></li>`
              `</ul>`